clang/test/Driver/sanitizer-ld.c

```diff
@@ -514,7 +514,7 @@
 // RUN:     -resource-dir=%S/Inputs/resource_dir \
 // RUN:     --sysroot=%S/Inputs/basic_linux_tree \
 // RUN:   | %{filecheck} --check-prefix=CHECK-UBSAN-LINUX-LINK-CXX
-// CHECK-UBSAN-LINUX-LINK-CXX: "{{.*}}ld"
+// CHECK-UBSAN-LINUX-LINK-CXX: "{{.*}}ld{{(.exe)?}}"
```

Comment:
The "CHECK" line has "LINUX" in the name. Are these tests only supposed to be run on Linux?

Reply:
Clang is always a cross-compiler. These tests are targeting linux, but can be run on any host.